usr/include/pkcs11types.h

```diff
@@ -19,6 +19,15 @@
 #ifndef _PKCS11TYPES_H_
 #define _PKCS11TYPES_H_
 
+#include <dlfcn.h>
```

Comment:
the pkcs11types.h is an external header that gets installed on the target system and gets included in user applications. I think we should not do these things there, as this may interfere with the user application's defines. 
Also, dlfcn.h might not always be needed by the user application, so please don't include this here.

I guess this rather belongs into `defs.h` in common, or similar.